cachish/backends/hawk_json_http.py

```diff
@@ -0,0 +1,45 @@
+from mohawk import Sender
+from .json_http import JsonHttp
+
+class Hawk(JsonHttp):
+    def __init__(self, #hawk secret id, secret key, algorithm
+            secret_id=None,
+            secret=None,
+            algorithm='sha256',
+            url=None,
+            field=None
+            ): # pylint: disable=too-many-arguments
+        assert secret_id is not None
+        assert secret is not None
+        assert algorithm is not None
+
+        self.secret_id = secret_id
+        self.secret = secret
+        self.algorithm = algorithm
+        super(Hawk, self).__init__(url=url, field=field) #sets self.target_url
+
+    @property
+    def tag(self):
+        return 'Hawk/%s' % self.target_url
+
+    #dictionary stuff with headers?
+    def get(self):
+        sender = Sender({
+                'id': self.secret_id,
+                'key': self.secret,
+                'algorithm': self.algorithm,
+            },
+            self.target_url,
+            'GET',
+            '',
```

Comment:
While the other fields are fairly self-explanatory, these two last ones are rather cryptic. Why is an empty string passed? These are better used as keyword-arguments, so that's clear what they are, and you can probably add a comment with a link to the mohawk docs where they mention that this is the correct way to set the headers for a GET request.